keys.go

```diff
@@ -217,11 +234,13 @@ func (c *Client) createKey(ctx context.Context, key Key) (*Key, error) {
 		Keys: []Key{key},
 	}
 
-	req, err := c.newRequest("POST", "keys", &keysRequest)
+	req, err := c.newRequest(http.MethodPost, path, &keysRequest)
 	if err != nil {
 		return nil, err
 	}
-
+	if path == keysWithPolicyOverridesPath {
```

Comment:
We should not force full representation unless the user is explicity asking for it.